generators/server/templates/src/test/java/package/config/_WebConfigurerTest.java

```diff
@@ -146,7 +146,7 @@ public void testCustomizeServletContainer() {
 
     @Test
     public void testCustomizeServletContainerNotProd() {
-        UndertowEmbeddedServletContainerFactory container = new UndertowEmbeddedServletContainerFactory();
+        UndertowServletWebServerFactory container = new UndertowServletWebServerFactory();
         webConfigurer.customize(container);
         assertThat(container.getMimeMappings().get("abs")).isEqualTo("audio/x-mpeg");
         assertThat(container.getMimeMappings().get("html")).isEqualTo("text/html;charset=utf-8");
```

Comment:
MediaType.TEXT_HTML_VALUE + StandardCharsets.UTF_8.name()

Reply:
Damn, too late: I will make a PR for that

Reply:
Thanks @Tcharl -> this code can definitely be improved, I did that quickly and with very little documentation - my main objective was to release something that works, so that we can then have PRs to improve